static/public/javascripts/colorPalettes.js

```diff
@@ -0,0 +1,28 @@
+'use strict';
+
+// const COLORS_7 = [
+//     '#00935e', '#50b67f', '#9fd99f', '#effcc0', '#FBD989', '#e29e6d', '#c96251'
+// ].reverse();
+
+
+// const COLORS_3 = [
```

Comment:
Should we uncomment and export COLORS_3 and COLORS_5 ? Are they needed in the code somewhere ?